### PR TITLE
Standarize Gaussian distribution input.

### PR DESCRIPTION
**Pull request by cmbiwer**
Makes the ``Gaussian`` distribution similar to the other distributions ``__init__`` in ``distributions.py``.

For Gaussian with bounds 1.4 and 1.5 and mean 1.3 and variance 0.01 do:
```
gauss = Gaussian(mass1=(1.3, 1.5, 1.4, 0.01))
```

This example is also in the docstring.

### CODE REVIEW

File: pycbc/inference/distributions.py [hunk 1]
```diff
@@ -862,25 +862,22 @@ class Gaussian(Uniform):
...
     name : 'guassian'
         The name of this distribution.
+
+    Example
+    -------
+    >> dist = Gaussian(mass1=(1.3,1.5,1.4,0.01)
     """
     name = "gaussian"
-    def __init__(self, variable_args, low, high, mean, var, **kwargs):
+    def __init__(self, **params):
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E301] expected 1 blank line, found 0

File: pycbc/inference/distributions.py [hunk 2]
```diff
@@ -1036,15 +1033,17 @@ def from_config(cls, cp, section, variable_args):
...
-        var = []
+        params = {}
         for param in variable_args:
-            low.append( float(cp.get_opt_tag(section, "min-%s"%param, tag)) )
-            high.append( float(cp.get_opt_tag(section, "max-%s"%param, tag)) )
-            mean.append( float(cp.get_opt_tag(section, "mean-%s"%param, tag)) )
-            var.append( float(cp.get_opt_tag(section, "var-%s"%param, tag)) )
+            params[param] = (None, None, None, None)
+            params[param][0] = float(cp.get_opt_tag(section,
+                                                   "min-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 3]
```diff
@@ -1036,15 +1033,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][1] = float(cp.get_opt_tag(section,
+                                                   "max-%s"%param, tag))
+            params[param][2] = float(cp.get_opt_tag(section,
+                                                   "mean-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 4]
```diff
@@ -1036,15 +1033,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][3] = float(cp.get_opt_tag(section,
+                                                   "var-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 5]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
-        var = []
+        params = {}
         for param in variable_args:
-            low.append( float(cp.get_opt_tag(section, "min-%s"%param, tag)) )
-            high.append( float(cp.get_opt_tag(section, "max-%s"%param, tag)) )
-            mean.append( float(cp.get_opt_tag(section, "mean-%s"%param, tag)) )
-            var.append( float(cp.get_opt_tag(section, "var-%s"%param, tag)) )
+            params[param] = (None, None, None, None)
+            params[param][0] = float(cp.get_opt_tag(section,
+                                                   "min-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 6]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][1] = float(cp.get_opt_tag(section,
+                                                   "max-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 7]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][2] = float(cp.get_opt_tag(section,
+                                                   "mean-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 8]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][3] = float(cp.get_opt_tag(section,
+                                                   "var-%s"%param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

File: pycbc/inference/distributions.py [hunk 9]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
-        var = []
+        params = {}
         for param in variable_args:
-            low.append( float(cp.get_opt_tag(section, "min-%s"%param, tag)) )
-            high.append( float(cp.get_opt_tag(section, "max-%s"%param, tag)) )
-            mean.append( float(cp.get_opt_tag(section, "mean-%s"%param, tag)) )
-            var.append( float(cp.get_opt_tag(section, "var-%s"%param, tag)) )
+            params[param] = (None, None, None, None)
+            params[param][0] = float(cp.get_opt_tag(section,
+                                                   "min-%s" % param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent

File: pycbc/inference/distributions.py [hunk 10]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][1] = float(cp.get_opt_tag(section,
+                                                   "max-%s" % param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent

File: pycbc/inference/distributions.py [hunk 11]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][2] = float(cp.get_opt_tag(section,
+                                                   "mean-%s" % param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent

File: pycbc/inference/distributions.py [hunk 12]
```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
...
+            params[param][3] = float(cp.get_opt_tag(section,
+                                                   "var-%s" % param, tag))
```
Comment (ahnitz) [inline pycbc/inference/distributions.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent

**Comment by stevereyes01** [comment]
Thanks for this Chris.

**Pull request merged**